redis/reply_test.go

```diff
@@ -89,9 +89,9 @@ var replyTests = []struct {
 		ve([]string{"v1", "v2"}, nil),
 	},
 	{
-		"strings([string, string])",
-		ve(redis.Strings([]interface{}{"v1", "v2"}, nil)),
-		ve([]string{"v1", "v2"}, nil),
+		"stringMap([string, string])",
+		ve(redis.StringMap([]interface{}{[]byte("key1"), []byte("v1"), []byte("key2"), []byte("v2")}, nil)),
+		ve(map[string]string{"key1": "v1", "key2": "v2"}, nil),
```

Comment:
This looks like a mistake, as it's replacing a `Strings` test with `StringMap` test and not adding `BytesMap`